website/content/docs/secrets/databases/index.mdx

```diff
@@ -104,75 +105,157 @@ TTL.
+![Partial screenshot of the Vault GUI showing the "Enable database" page](/img/gui/databases/enableDatabase.png)
 
-   This secrets engine can configure multiple database connections. For details
-   on the specific configuration options, please see the database-specific
-   documentation.
+1. Open the web UI for your Vault instance.
+1. Login under the target namespace or choose the target namespace from the
+   selector at the bottom of the left-hand menu and re-authenticate.
+1. Select **Secrets Engines** from the left-hand menu.
+1. Click **Enable new engine +** on the **Secrets Engines** page.
+1. Select **Databases**.
+1. Click **Enable engine**.
+1. Click **Save** to enable the plugin.
 
-1. After configuring the root user, it is highly recommended you rotate that user's
-   password such that the vault user is not accessible by any users other than
-   Vault itself:
+</Tab>
 
-   ```shell-session
-   $ vault write -force database/rotate-root/my-database
-   ```
+</Tabs>
 
-   !> When this is done, the password for the user specified in the previous step
-   is no longer accessible. Because of this, it is highly recommended that a
-   user is created specifically for Vault to use to manage database
-   users.
+<Tabs>
 
-1. Configure a role that maps a name in Vault to a set of creation statements to
-   create the database credential:
+<Tab heading="CLI" group="cli">
 
-   ```shell-session
-   $ vault write database/roles/my-role \
-       db_name=my-database \
-       creation_statements="..." \
-       default_ttl="1h" \
-       max_ttl="24h"
-   Success! Data written to: database/roles/my-role
-   ```
+Configure Vault with the proper plugin and connection information:
+
+```shell-session
+$ vault write database/config/my-database \
+    plugin_name="..." \
+    connection_url="..." \
+    allowed_roles="..." \
+    username="..." \
+    password="..." \
+```
+
+~> It is highly recommended a user within the database is created
+specifically for Vault to use. This user will be used to manipulate
+dynamic and static users within the database. This user is called the
+"root" user within the documentation.
+
+Vault will use the user specified here to create/update/revoke database
+credentials. That user must have the appropriate permissions to perform
+actions upon other database users (create, update credentials, delete, etc.).
+
+This secrets engine can configure multiple database connections. For details
+on the specific configuration options, please see the database-specific
+documentation.
+
+</Tab>
+
+<Tab heading="GUI"  group="gui">
+
+![Partial screenshot of the Vault GUI showing the "Create connection" page](/img/gui/databases/configureDBConnection.png)
+
+1. From Secret Engines List, click on **database/**
+1. Click on **Connections** tab and click **Create connection +**
+1. Click on Database plugin dropdown and select desired plugin ie. **postgreSQL**
```

Comment:
```suggestion
1. Click on the **Database plugin** dropdown and select your database.
```
Given that it's just a dropdown, I'm not sure it's worth calling out an example value.